numpy/random/tests/test_extending.py

```diff
@@ -40,10 +40,8 @@
 
 @pytest.mark.skipif(cython is None, reason="requires cython")
 @pytest.mark.slow
-@pytest.mark.skipif(sys.platform == 'win32', reason="cmd too long on CI")
 def test_cython(tmp_path):
     examples = os.path.join(os.path.dirname(__file__), '..', '_examples')
-    base = os.path.dirname(examples)
     shutil.copytree(examples, tmp_path / '_examples')
```

Comment:
This should become `str(tmp_path / '_examples')`

Reply:
OK, will make the change.